pallets/automation-time/src/tests.rs

```diff
@@ -1818,15 +1834,6 @@ mod run_dynamic_dispatch_action {
 				encoded_call,
 				task_id.clone(),
 			);
-
-			assert_eq!(
-				events(),
-				[RuntimeEvent::AutomationTime(crate::Event::DynamicDispatchResult {
-					who: account_id,
-					task_id: task_id.clone(),
-					result: Err(DispatchError::from(frame_system::Error::<Test>::CallFiltered)),
-				}),]
-			);
```

Comment:
Where’s the replacement of this error verification? What is this case testing without it?

Reply:
The same applies to the below tests.

Reply:
Fixed